api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -504,6 +511,22 @@ message Cluster {
       google.protobuf.UInt32Value hash_balance_factor = 2 [(validate.rules).uint32 = {gte: 100}];
     }
 
+    // Configuration for :ref:`slow start mode <arch_overview_load_balancing_slow_start>`.
+    message SlowStartConfig {
+      // Size of slow start window in seconds, default value is 0.
+      // If set to value greater than 0, the newly created host remains in slow start mode starting from it's creation time
+      // for the duration of slow start window.
+      google.protobuf.UInt32Value slow_start_window = 1 [(validate.rules).uint32 = {gte: 0}];
```

Comment:
If the default is 0, this can just be a basic type?

Reply:
My comment was actually wrong, default value in code is 1.0.

Reply:
changed